### DISCUSSION TITLE
Does Vue Flow support expanding and clapsing children nodes?

### DISCUSSION THREAD

**Discussion by WilliamYe79** [bcakmakoglu/vue-flow#1436] [Q&A] [answered]
Hi,
I just found Vue Flow today, and the effects look really beautiful! I am conceiving to add a customized mind map into my application. However, in normal mind maps, at the border of a node, we can show a '+' sign, which can be clicked to expand (to show) all the child nodes of a node; we can also show a '-' sign, which can be clicked to clapse (to hide) all the child nodes of it. Does Vue Flow provide such kind of functionalities? If yes, would you tell me what documents I should read to get some idea about this functionality and how to use it in the code?

**Comment by bcakmakoglu** [comment, marked as answer]
Read the answer here: https://github.com/bcakmakoglu/vue-flow/discussions/1391

Although in your case I don't think you need *nodes* inside of nodes. Just use a list of items inside of a custom node and collapse/expand that list.
